### PR TITLE
[proposal] retry enabling coin to fa migration

### PR DESCRIPTION
**Pull request by lightmark**
retry coin to fa migration.
This PR redo https://github.com/aptos-foundation/mainnet-proposals/pull/74

### CODE REVIEW

File: metadata/2024-07-23-enable-fa-migration/enable_coin_to_fa_migration.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Enable the mapping from coin to fungible asset, including APT",
+  "description": "Enables the changes in AIP-63 https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-63.md",
+  "source_code_url": "https://github.com/aptos-labs/aptos-core",
```
Comment (sherry-x) [inline metadata/2024-07-23-enable-fa-migration/enable_coin_to_fa_migration.json:4]: source code url should be the .move file in this repo, the script for this proposal

Comment (lightmark) [reply metadata/2024-07-23-enable-fa-migration/enable_coin_to_fa_migration.json:4]: let me change

**Review by vgao1996 — APPROVED** [review]

File: metadata/2024-07-24-enable-coin-to-fa-migration/enable_coin_to_fa_migration.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Enable the mapping from coin to fungible asset, including APT",
+  "description": "Enables the changes in AIP-63 https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-63.md",
+  "source_code_url": "https://github.com/aptos-foundation/mainnet-proposals/tree/main/sources/2024-07-23-enable-fa-migration",
```
Comment (sherry-x) [inline metadata/2024-07-24-enable-coin-to-fa-migration/enable_coin_to_fa_migration.json:4]: update source url, 07-23 -> 07-24

**Review by sherry-x — APPROVED** [review]

File: sources/2024-07-24-enable-coin-to-fa-migration/0-features.move [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+//
+script {
+    use aptos_framework::aptos_governance;
+    use std::features;
+
+    fun main(proposal_id: u64) {
+        let framework_signer = aptos_governance::resolve_multi_step_proposal(
+            proposal_id,
+            @0x1,
+            x"c6a11ff00d3494f29052baf579e2b38c0eae949fb5da97d0ab68379ef300904e",);
```
Comment (sherry-x) [inline sources/2024-07-24-enable-coin-to-fa-migration/0-features.move:14]: this is the hash for this script?

Comment (lightmark) [reply sources/2024-07-24-enable-coin-to-fa-migration/0-features.move:14]: yes

**Pull request merged**